docs/screenshots/README.md

```diff
@@ -0,0 +1,14 @@
+### Main Screen
+![Main-Screen](https://raw.githubusercontent.com/fabianherrera/roger_ui/develop/docs/screenshots/main.png)
```

Comment:
Shouldn't the screenshots be updated to show the latest UI?

Reply:
done. All screenshots and their links to bettyblock organization were included.